Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -5772,6 +5772,10 @@ static BOOL shouldUseHighlightsForMarkedText(NSAttributedString *string)
 {
     if (m_warningView)
         return;
+#if ENABLE(SCREEN_TIME)
+    if (m_view && [[m_view _screenTimeWebpageController] URLIsBlocked])
```

Comment:
`m_view && ` is unnecessary. `[[m_view _screenTimeWebpageController] URLIsBlocked]` will evaluate to `nil` if `m_view` is `nil`.